canvas_modules/common-canvas/locales/command-actions/locales/en.json

```diff
@@ -16,6 +16,11 @@
 	"action.createSuperNode": "Create {node_label} supernode",
 	"action.deconstructSuperNode": "Deconstruct {node_label} supernode",
 	"action.deleteLink": "Delete link",
```

Comment:
General comment: if you add new messages to the English translation file (like these messages below) you need to added corresponding messages to the Esperanto file (the one called eo.json). As you will see the messages in that file have the same text but with prefix and suffix characters. The Esperanto file is used for testing to see if translated messages and text, that are longer that English text, can be displayed OK in the interface. You will need to replicate the new messages into the Esperanto file and add some prefix and suffix characters. The number of prefix and suffix characters is roughly proportional to the length of the English text. That is, a short English message would have a small number of extra characters and a long English message would have a larger number of characters. Please ping me on Slack if this is not clear.